pdk.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.1.9'
 
   spec.add_runtime_dependency 'bundler', '~> 1.15'
-  spec.add_runtime_dependency 'cri', '~> 2.9.1'
+  spec.add_runtime_dependency 'cri', '~> 2.10.1'
```

Comment:
This is throw-in. I'd prefer if this was taken out and a new ticket created to update CRI, so we can update the vanagon component at the same time.